src/rashdf/plan.py

```diff
@@ -1317,3 +1435,89 @@ def cross_sections_additional_velocity_total(self) -> DataFrame:
             A DataFrame containing the velocity inside the cross sections
         """
         return self.steady_profile_xs_output(XsSteadyOutputVar.VELOCITY_TOTAL)
+
+    def _zmeta(self, ds: xr.Dataset) -> Dict:
+        from kerchunk.hdf import SingleHdf5ToZarr
+        import zarr
+        import base64
+
+        encoding = {}
+        chunk_meta = {}
+        for var, da in ds.data_vars.items():
+            hdf_ds_path = da.attrs["hdf_path"]
+            hdf_ds = self.get(hdf_ds_path)
+            if hdf_ds is None:
+                continue
+            filters = SingleHdf5ToZarr._decode_filters(None, hdf_ds)
+            encoding[var] = {"compressor": None, "filters": filters}
+            storage_info = SingleHdf5ToZarr._storage_info(None, hdf_ds)
+            for key, value in storage_info.items():
+                chunk_key = f"{var}/{key[0]}.{key[1]}"
+                chunk_meta[chunk_key] = [str(self._loc), value["offset"], value["size"]]
+        zarr_tmp = zarr.MemoryStore()
+        ds.to_zarr(zarr_tmp, mode="w", compute=False, encoding=encoding)
+        zarr_meta = {"version": 1, "refs": {}}
```

Comment:
Whats the version of 1 here refer to? Im guessing thats just a zarr metadata version that will only be updated when changes are made specifically to the zarr metadata functions?

Reply:
Yeah, it's the Zarr metadata version. More details here: https://fsspec.github.io/kerchunk/spec.html